wallet/file/file.go

```diff
@@ -279,6 +280,7 @@ func (af *fileAccountFactory) DataValidator(kind wallet.ImportKind, rawCfg map[s
 				}
 			case wallet.AlgorithmSecp256k1Raw:
 				// Ensure the private key is hex encoded.
+				ans = af.DataTransformer(kind, rawCfg)(ans)
```

Comment:
Why add this `DataTransformer` stuff when you could just do the two-line transformation here?

Reply:
Because validator does not transform the answer - it only validates. As I understand we want to trim 0x from the answer or not?

Reply:
Ah didn't see that this is the validator function.

Looking now at `Import` itself, it seems that the `0x` prefix is already supported by `Secp256k1FromHex` that is used during the actual import? So instead of introducing `DataTransformer` just change this validation to use `Secp256k1FromHex` instead of `hex.DecodeString` and it should work? Also update the error message below.

Reply:
Nice. Ok, will do it like this, but this will not trim the `0x`.